assets/src/edit-story/components/draganddrop/index.js

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { useUploader } from '../../app/uploader';
+import { ReactComponent as UploadIcon } from './icons/upload.svg';
```

Comment:
Unfortunately not. Because we've switched to svgr (which is better), the default import is now the file path as a string - and you have to do named imports to get the component. It's a bit annoying, but [can't be helped](https://github.com/google/web-stories-wp/pull/244#issuecomment-583114369).

Reply:
Well, #244 hasn't been merged yet though.

Reply:
I did as I saw the change in #244  . I hoped it would land before this PR went in. But if it doesn't then I am willing to change it back to the way I had it before.

Reply:
Now that #244 is merged, making as resolved.